source/fundamentals/crud/write-operations/upsert.txt

```diff
@@ -4,8 +4,107 @@ Insert or Update in a Single Operation
 
 .. default-domain:: mongodb
 
-If your application stores and modifies data in MongoDB, you probably use
-insert and update operations. In certain workflows, you may need to choose
-between an insert and update depending on whether the document exists.
-In these cases, you can streamline your application logic by using the
-``upsert`` option available in the following methods:
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+.. _upsert_golang:
+
+Overview
+--------
+
+In this guide, you can learn how to specify an upsert.
```

Comment:
Suggestion/Not necessary for LGTM: 

This overview is a bit too minimal + uses a technical term, "upsert", without defining it or linking out to a description. 

Can you update the overview to describe the page a bit more in-depth and include a description or link to upsert?
For instance: 

"In this guide, you will learn to perform an upsert operation that updates documents which match a query filter or inserts a document if there are no matching documents".

Reply:
Agreed! I like how I currently format the page, so I'll link to the definition in the section right after.